tsl/src/continuous_aggs/materialize.c

```diff
@@ -665,12 +791,19 @@ scan_update_invalidation_threshold(TupleInfo *ti, void *data)
 	Form_continuous_aggs_invalidation_threshold form =
 		(Form_continuous_aggs_invalidation_threshold) GETSTRUCT(new_tuple);
 	if (form->watermark > new_threshold)
-		elog(ERROR, "Internal Error: new invalidation threshold must not be less than the old one");
-	form->watermark = new_threshold;
+		elog(INFO,
```

Comment:
I don't think we should do the `ts_catalog_update` in this case.

Reply:
changed to do this conditionally